DOC.md

```diff
@@ -3,48 +3,14 @@ SPDX-License-Identifier: MIT
 
 # Documentation of Interactive Configurator
 
-## API classes
+> Given the significant changes in the code and solving approach, this document will be divided into two sections, one explaining information that is still relevant and another, containing outdated information.
```

Comment:
This first paragraph is a bit unclear to me. Please rephrase it in such a way that it is clear that the following sections are all relevant to the current version, and that information on previous versions (please phrase it like this rather than "outdated information") will follow in the last section / in the bottom.